docs/api/FluxMixin.md

```diff
@@ -40,6 +40,17 @@ fluxMixin({
 });
 ```
 
+In some situations the data retrieved from one store depends on the state from another, you can use an array of store keys with a custom state getter to ensure the components state is updated when either store changes:
+
+```js
+fluxMixin(
+  ['posts', 'session'],
+  ([postStore, sessionStore]) => ({
```

Comment:
Maybe add a bit explaining that this is using array destucturing? Generally I expect most people interested in Flummox will be able to read ES6 but combined with the arrow function and the implicit return this could be confusing for ES6 newbies.